infrastructure/main.tf

```diff
@@ -1,3 +1,19 @@
+provider "vault" {
+  //  # It is strongly recommended to configure this provider through the
+  //  # environment variables described above, so that each user can have
+  //  # separate credentials set in the environment.
+  //  #
+  //  # This will default to using $VAULT_ADDR
+  //  # But can be set explicitly
+  address = "https://vault.reform.hmcts.net:6200"
+}
+
+data "vault_generic_secret" "sscs_notify_api_key" {
+  path = "secret/test/sscs/sscs_notify_api_new_key"
+}
+
+
+
```

Comment:
Can we have removed too much of empty lines only? Everything else seems to be fine